docs/source/AdministratorGuide/Resources/Storages/index.rst

```diff
@@ -0,0 +1,242 @@
+        Access = remote
+        SpaceToken = LHCb-EOS
+        WSUrl = /srm/v2/server?SFN=
+      }
+    }
+
+
+This definition would be strictly equivalent to::
+
+    StorageElementBases
+    {
+      CERN-EOS
+      {
+        BackendType = Eos
+        SEType = T0D1
+        AccessProtocol.1
+        {
+          Host = srm-eoslhcb.cern.ch
+          Port = 8443
+          PluginName = GFAL2_SRM2
+          Protocol = srm
+          Path = /eos/lhcb/grid/prod
+          Access = remote
+          SpaceToken = LHCb-EOS
+          WSUrl = /srm/v2/server?SFN=
+        }
+      }
+    }
+    StorageElements
+    {
+      CERN-DST-EOS
+      {
+        BackendType = Eos
+        SEType = T0D1
+        AccessProtocol.1
+        {
+          Host = srm-eoslhcb.cern.ch
+          Port = 8443
+          PluginName = GFAL2_SRM2
+          Protocol = srm
+          Path = /eos/lhcb/grid/prod
+          Access = remote
+          SpaceToken = LHCb-EOS
+          WSUrl = /srm/v2/server?SFN=
+        }
+      }
+      CERN-USER
+      {
+        BackendType = Eos
+        SEType = T0D1
+        PledgedSpace = 205
+        AccessProtocol.1
+        {
+          Host = srm-eoslhcb.cern.ch
+          Port = 8443
+          PluginName = GFAL2_SRM2
+          Protocol = srm
+          Path = /eos/lhcb/grid/user
+          Access = remote
+          SpaceToken = LHCb_USER
+          WSUrl = /srm/v2/server?SFN=
+        }
+      }
+      GFAL2_XROOT
+      {
+        Host = eoslhcb.cern.ch
+        Port = 8443
+        PluginName =  GFAL2_XROOT
+        Protocol = root
+        Path = /eos/lhcb/grid/user
+        Access = remote
+        SpaceToken = LHCb-EOS
+        WSUrl = /srm/v2/server?SFN=
+      }
+    }
+
+Note that base SE must be separated from the inherited SE in two different sections. You can also notice that the name of the protocol section can be a plugin name. In this way, you do not need to specify a plugin name inside.
+
+
+
+
+Available protocol plugins
+--------------------------
+
+DIRAC comes with a bunch of plugins that you can use to interact with StorageElements.
+These are the plugins that you should define in the `PluginName` option of your StorageElement definition.
+
+  - DIP: used for dips, the DIRAC custom protocol (useful for example for DIRAC SEs).
+  - File: offers an abstraction of the local access as an SE.
+  - SRM2 (deprecated): for the srm protocol, using the deprecated gfal libraries.
+  - RFIO (deprecated): for the rfio protocol.
+  - Proxy: to be used with the StorageElementProxy.
+  - XROOT: for the xroot protocol, using the python xroot binding (http://xrootd.org/doc/python/xrootd-python-0.1.0/#).
+
+There are also a set of plugins based on the gfal2 libraries (https://dmc.web.cern.ch/projects).
+
+  - GFAL2_SRM2: for srm, replaces SRM2
+  - GFAL2_XROOT: for xroot, replaces XROOT
+  - GFAL2_HTTPS: for https
+  - GFAL2_GSIFTP: for gsiftp
+
+
+Default plugin options:
+
+* `Access`: `Remote` or `local`.
```

Comment:
Where these options go? And are they really "Remote" (capitalized) and "local" (not capitalized)?

Reply:
In fact they are not case sensitive, but I should be consistent, and add more doc. Will do